README.md

```diff
@@ -1,69 +1,121 @@
-[here](https://mampf.mathi.uni-heidelberg.de/media/384/play).
+To give you a closer look, here are some **screenshots** taken from our live system:
 
-MaMpf is equipped with a tagging system and rich visualisations for content relations,
-making use of [cytoscape.js](http://js.cytoscape.org/).
+<details>
+  <summary>Video player</summary>
+  
+  Try out the video player [here](https://mampf.mathi.uni-heidelberg.de/media/384/play) (even without any account). Press `i` to open the outline on the right. It can hold references to other parts of the video or other items in the whole MaMpf database. The player makes use of WebVTT and HTML5 video capabilities of modern browsers.
 
-![tags](public/tag_visualisation.png)
+  <a href="https://mampf.mathi.uni-heidelberg.de/media/384/play">
+    <img src="https://github.com/MaMpf-HD/mampf/assets/37160523/ff049eeb-3c25-4db0-a21e-efd51e566256" alt="MaMpf video player"/>
+  </a>
+</details>
 
-MaMpf has a quiz system that allows you to create complex quizzes quite easily.
+<details>
+  <summary>Courses overview</summary>
+  
+  Here, users can select courses from the current semester or from previous ones.
 
-![quizzes](public/quizzes.png)
+  ![User courses view](https://github.com/MaMpf-HD/mampf/assets/37160523/a1e386ad-7642-49f2-aecf-f2f0722cc3c1)
+</details>
 
-MaMpf makes use of the JS based symbolic math expression evaluator
-[nerdamer](https://github.com/jiggzson/nerdamer) to parse student's input in quizzes.
+<details>
+  <summary>Lectures overview</summary>
+  
+  In the lectures view, users can click on a lecture to see the video.
 
+  ![User lectures view](https://github.com/MaMpf-HD/mampf/assets/37160523/a3936d73-dc45-489d-85f8-68326f61654a)
+</details>
 
-For more information see this [blog](https://mampfdev.wordpress.com).
-There you can also find a [screenshot gallery](https://mampfdev.wordpress.com/gallery/).
-## System background
+<details>
+  <summary>Graph tag search</summary>
+  
+  MaMpf is equipped with a tagging system and rich visualizations for content relations, making use of [cytoscape.js](http://js.cytoscape.org/).
 
-[![MaMpf](https://img.shields.io/endpoint?url=https://dashboard.cypress.io/badge/simple/v45wg9/main&style=flat&logo=cypress)](https://dashboard.cypress.io/projects/v45wg9/runs)
-[![codecov](https://codecov.io/gh/MaMpf-HD/mampf/branch/main/graph/badge.svg?token=x7Zq3m5lVH)](https://codecov.io/gh/MaMpf-HD/mampf)
+  ![Search graph](https://github.com/MaMpf-HD/mampf/assets/37160523/cd54b651-70c0-439d-a8dd-01de95995cb5)
+</details>
 
-MaMpf is implemented in Ruby on Rails.
+<details>
+  <summary>Quizzes</summary>
 
-* Ruby version: 3.1.4
-* Rails Version: 7.0.4.3
-* Test suite: rspec, cypress
-* support for I18n
+  Users can play quizzes in MaMpf and get immediate feedback. In order to parse student's input in quizzes (e.g. when they enter a concrete number), MaMpf makes use of the JS based symbolic math expression evaluator [nerdamer](https://github.com/jiggzson/nerdamer).
```

Comment:
> e.g. when they enter a concrete number

Maybe rather go for "symbolic expression" in the example here, otherwise one could wonder why we need a CAS just to check if a number is entered correctly.